### PR TITLE
Feature/sign-in-restore-password

### PR DESCRIPTION
**Pull request by milkavladislav**

### CODE REVIEW

File: src/pages/ForgotPassword/Index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import { FieldValues, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useNavigate } from "react-router-dom";
+import Field from "components/DefaultField/Index";
+import { SIGN_IN_ROUTE } from "utils/routeConsts";
+import { useTranslation } from "react-i18next";
+import { PRIMARY } from "utils/colorConsts";
+import S from "./style";
+import ForgotPasswordSchema from "./schema";
+import api from "../../services/auth/forgotPassword";
```
Comment (lyudad) [inline src/pages/ForgotPassword/Index.tsx]: absolute path

File: src/pages/RestorePassword/Index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import { Button, Modal, Typography } from "antd";
+import { FieldValues, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useLocation, useNavigate } from "react-router-dom";
+import Field from "components/DefaultField/Index";
+import { HOME_PAGE_ROUTE } from "utils/routeConsts";
+import { useTranslation } from "react-i18next";
+import S from "./style";
+import RestorePasswordSchema from "./schema";
+import api from "../../services/auth/forgotPassword";
```
Comment (lyudad) [inline src/pages/RestorePassword/Index.tsx]: same

File: src/pages/ForgotPassword/Index.tsx (hunk 1 above)
Comment (milkavladislav) [reply src/pages/ForgotPassword/Index.tsx]: done

File: src/pages/RestorePassword/Index.tsx (hunk 2 above)
Comment (milkavladislav) [reply src/pages/RestorePassword/Index.tsx]: done

**Pull request merged**